examples/servant-elm-street-main/stack.yaml

```diff
@@ -0,0 +1,9 @@
+resolver:
+  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/18/23.yaml
+
+packages:
+- .
+
+extra-deps:
+- /home/magdalena/projects/servant-playground/servant-elm-street
```

Comment:
Use a relative path so that
* this works for others
* it doesn't contain information about your computer

Reply:
done